baseapp/middleware.go

```diff
@@ -90,3 +101,34 @@ func AccessHandler(f AccessCallback) func(next http.Handler) http.Handler {
 		})
 	}
 }
+
+var DefaultOtelFilter = filters.None(
+	filters.PathPrefix("/ping"),
+	filters.PathPrefix("/api/ping"),
+	filters.PathPrefix("/health"),
+	filters.PathPrefix("/api/health"),
+	filters.PathPrefix("/debug"),
+	filters.PathPrefix("/api/debug"),
+	filters.PathPrefix("/pprof"),
+	filters.PathPrefix("/api/pprof"),
+)
+
+func NewOtelHandler() func(http.Handler) http.Handler {
```

Comment:
I'd call this`NewTelemetryHandler` or `TelemetryHandler` to make the purpose more clear, even if OpenTelemetry is the implementation.